src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/application_gateway/waf_policy/_create.py

```diff
@@ -127,38 +126,37 @@ def _build_arguments_schema(cls, *args, **kwargs):
         match_conditions.Element = AAZObjectArg()
 
         _element = cls._args_schema.custom_rules.Element.match_conditions.Element
-        _element.match_values = AAZListArg(
-            options=["match-values"],
-            help="Match value.",
-            required=True,
+        _element.values = AAZListArg(
+            options=["values"],
+            help="Space-separated list of values to match.",
         )
-        _element.match_variables = AAZListArg(
-            options=["match-variables"],
-            help="List of match variables.",
+        _element.variables = AAZListArg(
+            options=["variables"],
+            help="Space-separated list of variables to use when matching. Variable values: RemoteAddr, RequestMethod, QueryString, PostArgs, RequestUri, RequestHeaders, RequestBody, RequestCookies.",
             required=True,
         )
-        _element.negation_condition = AAZBoolArg(
-            options=["negation-condition"],
-            help="Whether this is negate condition or not.",
+        _element.negate = AAZBoolArg(
+            options=["negate"],
+            help="Match the negative of the condition.",
```

Comment:
It seems the options changed

Reply:
Yes, but they haven't been exposed to the users (in create) or zero used (in update).